### PR TITLE
Feature newtonsolver

### PR DESCRIPTION
**Pull request by paulstapor**
Preequilibration is now possible by setting an option.

### CODE REVIEW

File: include/rdata.h [hunk 1]
```diff
@@ -132,8 +132,11 @@ class ReturnData {
     /** number of linear steps by Newton step for steady state problem */
     double *newton_numlinsteps;
 
-    /** steady state found be Newton solver */
-    double *xss;
+    /** preequilibration steady state found be Newton solver */
```
Comment (dweindl) [inline include/rdata.h]: "by"

File: include/rdata.h [hunk 2]
```diff
@@ -132,8 +132,11 @@ class ReturnData {
...
+    double *x0;
+    
+    /** preequilibration sensitivities found be Newton solver */
```
Comment (dweindl) [inline include/rdata.h]: "by"

File: include/rdata.h (hunk 1 above)
Comment (dweindl) [reply include/rdata.h]: Will that be clear to the user that this x0 is not his supplied initial states?

**Comment by FFroehlich** [comment]
I would generally recommend fixing things that were critized, even if the pull request was approved ...

**Pull request merged**